app/utils.py

```diff
@@ -32,6 +34,25 @@ def parse_phone_number(phone_number):
     return stripped
 
 
+def timedelta_to_minutes(duration):
+    """Convert timedelta object to minutes"""
+    minutes = duration.seconds / 60
+
+    return minutes
+
+
+def minutes_to_timedelta(minutes):
+    return timedelta(minutes=minutes)
+
+
+def flash_errors(form):
```

Comment:
where else would you suggest putting these? and flash_errors will show all of the errors at the top of the form at once, rather than showing them in succession. i'll add a docstring